pkg/validations/createcluster/createcluster.go

```diff
@@ -0,0 +1,86 @@
+package createcluster
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/gitops/flux"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/providers"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+type validationManager struct {
+	clusterSpec    *cluster.Spec
+	provider       providers.Provider
+	gitOpsFlux     *flux.Flux
+	cmdValidations interfaces.Validator
+	dockerExec     validations.DockerExecutable
+}
+
+func NewValidationManager(clusterSpec *cluster.Spec, provider providers.Provider, gitOpsFlux *flux.Flux, cmdValidations interfaces.Validator, dockerExec validations.DockerExecutable) *validationManager {
+	return &validationManager{
+		clusterSpec:    clusterSpec,
+		provider:       provider,
+		gitOpsFlux:     gitOpsFlux,
+		cmdValidations: cmdValidations,
+		dockerExec:     dockerExec,
+	}
+}
+
+func (v *validationManager) ValidateCreate(ctx context.Context) error {
```

Comment:
I think `Validate` is enough, since this is running in a validation manager in the validations/createcluster package

Reply:
Perfect! Updated this